locales/en-US.json

```
@@ -208,7 +209,10 @@
   "role": "Role",
   "save": "Save",
   "saved": "Saved",
-  "search": "Search",
+  "search": {
+    "search": "search",
```

```suggestion
    "search": "Search",
```

IMO, it's weird not having this capitalized